requirements/pytorch/base.txt

```diff
@@ -3,6 +3,7 @@
 
 numpy>=1.17.2, <1.23.1
 torch>=1.9.*, <1.13.0
+lightning-lite  # actual version comes from src
```

Comment:
won't this attempt to pull from pypi?

Reply:
In install time it adds the same version as in the codebase so it will be replaced by `lightning-lite==1.8.0rc1` and then does not matter where it is taken...

Reply:
I think my suspicion was right. On master, we are now seeing that it tries to pull from PyPi

Reply:
I believe the root is here: https://github.com/Lightning-AI/lightning/pull/15435#discussion_r1009982968